src/coreclr/src/jit/lowerxarch.cpp

```diff
@@ -1100,141 +1100,8 @@ void Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
     ContainCheckHWIntrinsic(node);
 }
 
-union VectorConstant {
```

Comment:
This was just moved to https://github.com/dotnet/runtime/pull/36267/files#diff-62e3e74a242676427a24505cb3b1c209R321